ros_ign_interfaces/srv/ControlWorld.srv

```diff
@@ -0,0 +1,11 @@
+bool pause                        
```

Comment:
Do you mind documenting all the fields in the message? Let's try to do better than `ign-msgs` :grimacing: